code/modules/mob/living/silicon/robot/subtypes/lost_drone_vr.dm

```diff
@@ -41,7 +41,7 @@
 		prob(15);"corrupted",
 		prob(10);"bad")
 
-	var/vore_law = prob(50)		// 1/2 chance of having vore-related laws
+	var/vore_law = prob(80)		// 4/5 chance of having vore-related laws
```

Comment:
80% is probably a bit too much. I would go with 60% at most.

Reply:
resolved